### ISSUE TITLE
loguru dependency and DEBUG output

### ISSUE THREAD

**Issue by seishonagon** [RhetTbull/osxmetadata#2]
At first run, complains that loguru is not installed on my machine (pip install loguru solves that). Also: output of commands contains many DEBUG lines. is it possible to turn them off? it's hard to parse output in my python script ...

**Comment by RhetTbull** [comment]
Thanks for letting me know. I've removed the debug code and dependency on loguru.

**Comment by RhetTbull** [comment]
Also, you can set the environment variable LOGURU_LEVEL to change what output gets logged by loguru.  E.g. export LOGURU_LEVEL=CRITICAL will only show critical log items, (e..g no DEBUG).  The current version of the code in github doesn't have any of the loguru code in it so you should not see any more debug output if you use the current version from github or PyPi.
